R/ml_model_lda.R

```diff
@@ -0,0 +1,13 @@
+new_ml_model_lda <- function(pipeline, pipeline_model, model,
+                                dataset, formula, feature_names,
+                                call) {
+
+  vocabulary <- pipeline_model$stages[[2]]$vocabulary
+
+  new_ml_model_clustering(
+    pipeline, pipeline_model,
+    model, dataset, formula,
+    vocabulary = vocabulary,
+    subclass = "ml_lda_model"
```

Comment:
This should be `"ml_model_lda"`, since `ml_lda_model` is the class for the pipeline stage; see `new_ml_lda_model()`.